release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -50,6 +50,20 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-13-installation-and-upgrade"]
 === Installation and upgrade
 
+[id="ocp-4-13-installation-vsphere-8-support"]
+==== Support for VMware vSphere version 8.0 
+{product-title} {product-version} supports VMware vSphere version 8.0. You can continue to install an {product-title} cluster on VMware vSphere version 7.0 Update 2.
```

Comment:
@bscott-rh Just want to make sure you are in the loop about this if you need to be based on that ticket we worked on about support for vSphere 8

Reply:
If this statement is valid we might need to say 

> {product-title} {product-version} now supports VMware vSphere version 8.0.

because there was some confusion previously about when VSphere 8 is supported.

Reply:
My understanding is that vSphere 8.0 is now supported, but @bscott-rh can confirm. I'm not sure if the "now" is necessary, because the note resides in the 4.13 release notes guide.

Reply:
thanks for looping me in. vSphere 8 is supported for 4.12 and 4.13, so you're good to go here.

Reply:
Thanks, @bscott-rh .

Hi @opayne1 . You OK with me moving this PR to the merging stage?

Reply:
Yes, I think you are good to move to merge review @dfitzmau 😄